docs/visualization/visualization_api/communication_information.md

```diff
@@ -17,6 +18,11 @@ comm = comm[0]
 # or comm = app.get_communication('pub_node', 'sub_node', 'topic_name')
 ```
 
+Note
+
+1. The `output_notebook()` is needed only when running the script on Jupyter Notebook.
+2. The function `get_communications` returns a List and the function `get_communication` returns one communication handler.
```

Comment:
@takeshi-iwanari 
I adapted your comment.
https://github.com/tier4/CARET_doc/pull/95#discussion_r999196872
> It will be kind for readers why it returns list and which one (index) should be used.

I added explanation.